chapters/bizleg-case-studies/startups/yellr.md

```diff
@@ -0,0 +1,208 @@
+# yellr
+### report for your community 
+
+Table of Contents
+
+
+*Executive summary*
+
+*Background research*
+
+*Situational Analysis*
+
+*Objectives*
+
+*Action Plan*
+
+*Financial Data*
+
+## Executive Summary
+
+Who we are:
+
+    Yellr was created as a hacks/hackers project by a group of volunteer developers and journalists in association with WXXI.
+
+What we sell:
+
+    Yellr is a software platform and app for journalists to use to reach out and engage hyper-local communities by creating geo-fenced “assignments” which will notify users of questions that a journalist wishes to be answered by users. Users can anonymously post pictures, audio, videos, and text to WXXI, which can be curated by journalists to publish web stories.
```

Comment:
These show up as code blocks. Maybe it'd be better to have the subheaders ("what we sell," etc) as H3 or blockquotes and format this as normal text.